docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
 # Support guidelines
 
+This page explains the support mechanisms we provide.
+
 !!! warning
 
-    Under Construction
+    Before asking any help, search and read this documentation site carefully.
+    Also, understand and follow our [discussion guidelines](../contributing/discussion-guidelines/index.md) during your discussions.
+
+Choose appropriate resources depending on what kind of help you need:
+
+- Documentation sites
+  - Various information
+- GitHub Discussions
+  - Questions
+  - Unconfirmed bugs
+  - Feature requests
+  - Design discussions
+- GitHub Issues
+  - Confirmed bugs
+- Slack
+  - Instant messaging between contributors
+- ROS Discourse
+  - General topics that should be widely announced
+
+The following sections are detailed descriptions of each resource.
+
+## Documentation sites
+
+[Docs guide](docs-guide.md) shows the list of useful documentation sites.
+Visit them and see if there is any information related to your problem.
+
+Note that the documentation sites aren't always latest and perfect.
+If you find some information is wrong, unclear, or missing in Autoware docs, feel free to submit a pull request following our [contribution guidelines](../contributing/index.md).
+
+!!!warning
+
+    Since this documentation site is still under construction, there are some empty pages.
+
+## GitHub Discussions
+
+If you have a problem with Autoware, check existing issues and questions and search similar issues first.
+
+- [Issues](https://github.com/autowarefoundation/autoware/issues)
+
+  Note that Autoware has multiple repositories listed in [autoware.repos](https://github.com/autowarefoundation/autoware/blob/main/autoware.repos).  
+  It is recommended to search across the repositories.
+
+- [Questions](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+
+If no answer was found, create a new question thread [here](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
+If your question is not answered in a timely manner (about a week), mention maintainers and remind them.
```

Comment:
```suggestion
If your question is not answered in a timely manner (about a week), please mention the maintainers to remind them.
```

Reply:
Hmm, could you please describe the difference?

Reply:
Removed `please`, They remind the maintainers by mentioning them so I put `to` there instead of `and`. But I think both work. We can keep `and`

Reply:
The line says to "mention the maintainers", but which maintainers specifically (does it depend on the specific repo, or anything like that?).

Reply:
I meant general `Autoware community's maintainers`, but no specific person from the maintainers.

Reply:
When do I have to add `then` for `if` sentences? :thinking:

Reply:
@kenji-miyake That is a good question! It's a bit like using `なら` I think? 

I re-read that sentence to myself a few times (and looked for other examples online), and I think the `then` can be removed in this particular example to improve the flow of the sentence (apologies, as I saw that you had already made a commit for this earlier).

Reply:
No problem. Since Google's style guide also uses `then`, I think there is no problem.
https://developers.google.com/style

But later we can consider making the style consistent if necessary! :+1: